front/src/utils/uiCoreHelpers.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
+export const createStringSelectOptions = (opts: string[], labelPlaceholder?: string) => ({
```

Comment:
Should the function be more generic in case `options` are a bit more complex ?

Reply:
I can introduce a second function for a usage of non string cases where the options are objects ?

Reply:
Would have been better to be able to have one function but it seems complicated